ui/widgets/resizable.js

```diff
@@ -226,6 +226,9 @@ $.widget( "ui.resizable", $.ui.mouse, {
 			this._removeHandles();
 			this._setupHandles();
 			break;
+		case "aspectRatio":
+			this._aspectRatio = !!value;
```

Comment:
Ah, thanks for pointing that out. Looking at just the diff, I forgot about the two.